artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/management/AddressControl.java

```diff
@@ -135,4 +135,35 @@ String sendMessage(@Parameter(name = "headers", desc = "The headers to add to th
                       @Parameter(name = "durable", desc = "Whether the message is durable") boolean durable,
                       @Parameter(name = "user", desc = "The user to authenticate with") String user,
                       @Parameter(name = "password", desc = "The users password to authenticate with") String password) throws Exception;
+
+   /**
+    * Pauses all the consumers of this address.Messages are no longer delivered to all its queues consumers.
+    * Newly added consumers will be paused too until resume is called.
+    * @throws java.lang.Exception
+    */
+   @Operation(desc = "Pauses the queues bound to this address", impact = MBeanOperationInfo.ACTION)
+   void pause() throws Exception;
+
+   /**
+    * Pauses all the consumers of this address.Messages are no longer delivered to all its queues consumers.
+    * Newly added consumers will be paused too until resume is called.
+    * @throws java.lang.Exception
+    */
+   @Operation(desc = "Pauses the queues bound to this address", impact = MBeanOperationInfo.ACTION)
+   void pause(@Parameter(name = "persist", desc = "if true, the pause state will be persisted.") boolean persist) throws Exception;
+
+   /**
+    * Resume all the consumers of this address.Messages are no longer delivered to all its queues consumers.
```

Comment:
Can you update the doc too please. It should state it pauses the queues, not consumers.